mantis-control-plane/mantis-control-plane-core/src/main/java/io/mantisrx/server/master/resourcecluster/TaskExecutorAllocationRequest.java

```diff
@@ -16,17 +16,17 @@
 
 package io.mantisrx.server.master.resourcecluster;
 
-import io.mantisrx.runtime.MachineDefinition;
 import io.mantisrx.server.core.domain.JobMetadata;
 import io.mantisrx.server.core.domain.WorkerId;
+import io.mantisrx.server.core.scheduler.SchedulingConstraints;
 import lombok.AllArgsConstructor;
 import lombok.Value;
 
 @Value
 @AllArgsConstructor(staticName = "of")
 public class TaskExecutorAllocationRequest {
```

Comment:
nit: not related to this change - but I'm guessing this class need not be public?